tests/integration/targets/aws_eks_fargate_profile/aliases

```diff
@@ -0,0 +1,5 @@
+# reason: slow
+# Tests take around 25 minutes to run
+unsupported
```

Comment:
Do we need to mark them with unsupported? @markuman

Reply:
Hm, it's 25min really "slow"? I dunno.

Reply:
IMO 25 minutes would be "slow", but we should be able to have them as supported.  Unsupported wouldn't run in CI even when the module changes, so unless something else is wrong with them we would want to try to have them enabled.

Reply:
Ok, so i will do a commit to remove the unsupported mark and run the tests.